src/API/Site/Controllers/MerchantCenter/PhoneNumberController.php

```diff
@@ -0,0 +1,202 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Merchant;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\BaseController;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use Google\Service\ShoppingContent\Account;
+use Exception;
+use WP_Error;
+use WP_REST_Request as Request;
+use WP_REST_Response as Response;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class PhoneNumberController
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter
+ */
+class PhoneNumberController extends BaseController {
```

Comment:
missing `@since`

Reply:
We can just make this an auto-comment for all PRs 😆  3ea21e082dc607ecd8de0d73f2c938ce742806d7